pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractBaseDispatcher.java

```diff
@@ -189,6 +220,32 @@ public int filterEntriesForConsumer(Optional<EntryWrapper[]> entryWrapper, int e
         return totalEntries;
     }
 
+    private EntryFilter.FilterResult getFilterResult(FilterContext filterContext, Entry entry) {
+        EntryFilter.FilterResult result = EntryFilter.FilterResult.REJECT;
+        for (EntryFilter entryFilter : entryFilters) {
+            if (entryFilter.filterEntry(entry, filterContext) == EntryFilter.FilterResult.ACCEPT) {
+                result = EntryFilter.FilterResult.ACCEPT;
```

Comment:
We should return ACCEPT only if all the filters accepted the entry? it more like the FilterChain of jetty http://www.servlets.com/javadoc/javax/servlet/FilterChain.html